smtp/server.go

```diff
@@ -28,6 +31,12 @@ func (s *Server) Start() {
 	if err != nil {
 		panic(err.Error())
 	}
+
+	// set a timeout for the inbound server
```

Comment:
This is not needed. It should be set through `Server{}` initialization in `main.go`.